build.gradle.kts

```diff
@@ -34,10 +34,17 @@ dependencies {
     implementation("io.github.palexdev:virtualizedfx:$virtualizedfxVersion")
     implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:$coroutinesVersion")
     implementation("org.jetbrains.kotlinx:kotlinx-coroutines-javafx:$coroutinesVersion")
+    implementation("junit:junit:4.13.1")
```

Comment:
version should be set in gradle.properties

Reply:
also, maybe testImplementation?

Reply:
no, with `testImplementation` it is not building